sources/packages/backend/apps/api/src/route-controllers/program-year/_tests_/e2e/program-year.students.controller.getProgramYearById.e2e-spec.ts

```diff
@@ -0,0 +1,120 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import { ProgramYear } from "@sims/sims-db";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getStudentToken,
+  FakeStudentUsersTypes,
+} from "../../../../testHelpers";
+
+describe("ProgramYearStudentsController(e2e)-getProgramYearById", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  let programYear2001: ProgramYear;
+  let programYear2026: ProgramYear;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    appDataSource = dataSource;
+    const programYearRepo = appDataSource.getRepository(ProgramYear);
+
+    programYear2001 = {
+      programYear: "2001-2002",
+      programYearDesc:
+        "Study starting between August 01, 2001 and July 31, 2002",
+      formName: "SFAA2001-02",
+      active: false,
+      parentFormName: "SFAA2001-01-parent",
+      partnerFormName: "SFAA2001-01-partner",
+      programYearPrefix: "2001",
+    } as ProgramYear;
+
+    programYear2026 = {
+      programYear: "2026-2027",
+      programYearDesc:
+        "Study starting between August 01, 2026 and July 31, 2027",
+      formName: "SFAA2026-27",
+      active: true,
+      parentFormName: "SFAA2026-01-parent",
+      partnerFormName: "SFAA2026-01-partner",
+      programYearPrefix: "2026",
+    } as ProgramYear;
+
+    const programYear2001Promise = programYearRepo.save(programYear2001);
+    const programYear2026Promise = programYearRepo.save(programYear2026);
+
+    [programYear2001, programYear2026] = await Promise.all([
+      programYear2001Promise,
+      programYear2026Promise,
```

Comment:
Can we use a date in the past instead of 2026? I liked the idea of using 2001, for instance.